software/unicycler/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: unicycler
+description: Assembles bacterial genomes
+keywords:
+  - sort
```

Comment:
sort? 😛

Reply:
maybe better keywords and then ready to merge 👍

Reply:
I left the keyword created by `modules create`... Sort may become a popular `keyword` 😝